packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -61,7 +61,10 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
             </hub-order-tracking>
         </div>
     </div>
-    <h2 data-translate="manager_hub_dashboard_services"></h2>
+    <h2
+        data-ng-if="$ctrl.products.length"
```

Comment:
Why not encapsulate in a `div`?

Reply:
Looks like it would make more sense, yes

Reply:
yep sure